download_models/downloadOVMSModels.sh

```diff
@@ -65,10 +76,16 @@ getOVMSModelFiles() {
 
 if [ $segmentationModelDownloaded -eq 0 ]; then
     echo "download segmentation model..."
-    mkdir -p "$localSegmentationMdlDirName/FP16-INT8/1"
+    mkdir -p "$localPersonVehicleDetectionMdlDirName/FP16-INT8/1"
```

Comment:
why change this? segmentation model should be the same right?

Reply:
good catch, I made a mistake here